apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -117,16 +122,22 @@ const Auth: React.FC = () => {
 						{authRouteToHeader[pathname as ValueOf<RoutesWithHeader>]}
 					</h1>
 					<span className={styles["form-header__sub-text"]}>
+						{pathname === AppRoute.FORGOT_PASSWORD && (
```

Comment:
Decided to show both hint text and link to sign in on forgot password form because user might remember the password during the process and it would be nice to have a way back to sign in